src/app/screens/Receive/index.tsx

```diff
@@ -49,7 +50,8 @@ function Receive() {
   const [paid, setPaid] = useState(false);
   const [pollingForPayment, setPollingForPayment] = useState(false);
   const mounted = useRef(false);
-
+  const [isLightning, setLightning] = useState(true);
```

Comment:
I wonder if it would be easier to understand if this was renamed to receiveType. What do you think?

type ReceiveType = 'onchain' | 'lightning';